packages/thirdweb/src/wallets/in-app/native/helpers/constants.ts

```diff
@@ -19,13 +21,18 @@ export const COGNITO_USER_POOL_ID = "us-west-2_UFwLcZIpq";
 export const COGNITO_APP_CLIENT_ID = "2e02ha2ce6du13ldk8pai4h3d0";
 export const COGNITO_IDENTITY_POOL_ID =
   "us-west-2:2ad7ab1e-f48b-48a6-adfa-ac1090689c26";
-export const GENERATE_RECOVERY_PASSWORD_LAMBDA_FUNCTION =
+export const GENERATE_RECOVERY_PASSWORD_LAMBDA_FUNCTION_V1 =
   "arn:aws:lambda:us-west-2:324457261097:function:recovery-share-password-GenerateRecoverySharePassw-bbE5ZbVAToil";
+export const GENERATE_RECOVERY_PASSWORD_LAMBDA_FUNCTION_V2 =
+  "arn:aws:lambda:us-west-2:324457261097:function:lambda-thirdweb-auth-enc-key-prod-ThirdwebAuthEncKeyFunction";
+export const GENERATE_RECOVERY_PASSWORD_LAMBDA_FUNCTION =
+  GENERATE_RECOVERY_PASSWORD_LAMBDA_FUNCTION_V2;
 
 // TODO allow overriding domain
-export const DOMAIN_URL_2023 = "https://embedded-wallet.thirdweb.com";
+export const DOMAIN_URL_2023 = getThirdwebBaseUrl("inAppWallet");
```

Comment:
i meant to refactor this - but all these URLs need to be functions otherwise you set your thirdwebDomains but it doesnt take effect because this is a const that gets set once on js load

Reply:
When would thirdwebDomains change post load?

Reply:
issue is that both setThirdwebDomains and this call get called at the same time with no guarantee of the order :(

Reply:
Oof, I can make all these functions in a new PR

Reply:
Yeah separate pr